lighthouse-core/gather/connections/cri.js

```diff
@@ -36,20 +36,6 @@ class CriConnection extends Connection {
   connect() {
     return this._runJsonCommand('new')
       .then(response => this._connectToSocket(/** @type {LH.DevToolsJsonTarget} */(response)))
-      .catch(_ => {
-        // Compat: headless didn't support `/json/new` before m59. (#970, crbug.com/699392)
-        // If no support, we fallback and reuse an existing open tab
-        log.warn('CriConnection', 'Cannot create new tab; reusing open tab.');
```

Comment:
I want to say I've actually gotten this log message before in headful somehow, but I can't remember the circumstances.